### ISSUE TITLE
[Feature Request] Insert a reference to an existing tiddler

### ISSUE THREAD

**Issue by 0xnandhi** [sukima/vim-tiddlywiki#10]
Hi , 

Can you please support autocomplete and existing tiddler in "Insert" mode (or) a key bind to search list of tiddler and insert the same at the cursor location.

leader<ti> -
- prompt for user input
- search the file system for the tidder with the name - fuzzy search.
- user selects a particular tidder and click inser.
- the selected tiddler is placed at the cursor location as a link [[<tiddler name>]]

I feel , this is the only missing piece :)

**Comment by creinig** [comment]
Yes, that would be really useful - and I guess most of the required code already exists in the "edit tiddler" functionality.
Autocomplete on "[[" should be a natural way to do it, but I'd first have to read up on autocomplete for that. Some mapping to insert a link is even easier, and I can see usefulness for both (eg typing [[ on a mobile device is a pain).

No promise on when I can get to it though.

**Comment by creinig** [comment]
Added normal mode command to insert link. For autocomplete I first have to do more research.

**Comment by 0xnandhi** [comment]
Thanks for looking into it. :)

**Comment by creinig** [comment]
@0xnandhi : I added a user completion function (invoke with C-X C-U, with the cursor behind the "[["). It's not too fancy, but should cover the most important use cases.
